client/my-sites/site-settings/reading-rss-feed-settings/SyndicationFeedsSetting.tsx

```diff
@@ -0,0 +1,54 @@
+import { useTranslate } from 'i18n-calypso';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormSettingExplanation from 'calypso/components/forms/form-setting-explanation';
+import FormTextInput from 'calypso/components/forms/form-text-input';
+
+export const SYNDICATION_FEEDS_OPTION = 'posts_per_rss';
+
+type SyndicationFeedsSettingProps = {
+	value?: number;
+	onChange: ( event: React.ChangeEvent< HTMLInputElement > ) => void;
+	disabled?: boolean;
+};
+
+export const SyndicationFeedsSetting = ( {
+	value = 10,
+	onChange,
+	disabled,
+}: SyndicationFeedsSettingProps ) => {
+	const translate = useTranslate();
+	return (
+		<FormFieldset>
+			<FormLabel id={ `${ SYNDICATION_FEEDS_OPTION }-label` } htmlFor={ SYNDICATION_FEEDS_OPTION }>
+				{ translate( 'Syndication feeds' ) }
+			</FormLabel>
+			<div>
+				{ translate( 'Show the most recent {{field /}} items', {
+					comment:
+						'The field value is a number that defines how many posts are shown on the posts, or the archive, page at a time.',
```

Comment:
I wonder if we could clarify this comment a bit more. Maybe we could use a similar string as is displayed to the users.

```suggestion
						'The field value is a number of most recent posts that will be sent out at once to RSS feed subscribers.'
```